Exercises/Index.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+
+alert("Enter two numbers to see if the first is greater than the second");
+
+let firstNumber = prompt('Please enter your first number.',100);
+let secondNumber = prompt('Please enter your Second number.',100);
```

Comment:
Would be good to convert entered values into "number" type before comparing them.